pkg/collector/corechecks/servicediscovery/apm/detect_nix_test.go

```diff
@@ -19,25 +19,22 @@ func Test_javaDetector(t *testing.T) {
 	data := []struct {
 		name   string
 		args   []string
-		envs   []string
```

Comment:
I would recommend leaving this field in the anonymous testing struct (converted to map[string]string to match the new API).

Reply:
Done.